packages/rx_bloc_cli/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rx_bloc_cli
 description: "rx_bloc_cli that enables quick project setup including: flavors, localization [intl], state management [rx_bloc], routing [go_router], design system, analytics [firebase], tests"
-version: 4.0.0
+version: 5.1.0
```

Comment:
Don't forget to update the `rxBlocCliPackageVersion` inside the `packages/rx_bloc_cli/lib/src/rx_bloc_cli_constants.dart` file.

Reply:
Done, thanks!